democracy_club/settings/base.py

```diff
@@ -226,7 +226,7 @@ def blog_markdown(value):
 GOCARDLESS_REDIRECT_URL = "https://democracyclub.org.uk/donate/process/"
 
 SITE_TITLE = "Democracy Club"
-CANONICAL_URL = "https://democracyclub.org.uk"
+CANONICAL_URL = f"https://{os.environ.get('FQDN')}"
```

Comment:
You'll need to `import os` at the top of the file (which is why CI is failing https://app.circleci.com/pipelines/github/DemocracyClub/WhoCanIVoteFor/1489/workflows/9ab91967-79b3-4328-8b4f-e4f971abbde5/jobs/3394)